KinoG/KinoG/App/Flows/FilmsModule/View/Cells/FilmTypeTableViewCell.swift

```diff
@@ -3,40 +3,39 @@
 
 import UIKit
 
+// TODO: - Вынести в отдельный файл?
+enum FilmType: Int {
```

Comment:
```suggestion
enum FilmType: Int {
```
ниже спасти в контролер

Reply:
Используется в двух файлах, во вью и в презентере